lib/shared/widget/navigation_bar/scaffold_pattern.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+import 'package:warren_task_one/movements/view/movements_page.dart';
+import 'package:warren_task_one/portfolio/view/portfolio_page.dart';
+
+import '../../../portfolio/provider/provider.dart';
+import '../../utils/assets.dart';
+import 'bottom_nav_button.dart';
+
+class ScaffoldPatten extends StatefulHookConsumerWidget {
```

Comment:
isso aqui em teoria é uma bottom navigation bar, tu só construiu de um jeito mais complexo... mas no caso, ScaffoldPatten não é um nome intuitivo...
Da uma olhada na doc https://api.flutter.dev/flutter/material/BottomNavigationBar-class.html

Reply:
ajustado, mudei o nome para BottomNavButton, 

```suggestion
class BottomNavButton extends HookConsumerWidget {
```